src/monitor/utils/exceptions.ts

```diff
@@ -18,3 +18,25 @@ export class InvalidMonitoringStatusException extends Error {
     this.message = `Monitoria possui status bloqueante: ${status}`;
   }
 }
+
+export class ScheduledMonitoringNotFoundException extends Error {
+  constructor() {
+    super();
+    this.message = 'Não foi encontrado nenhum agendamento com o id informado.';
+  }
+}
+
+export class ScheduleNotPendingException extends Error {
+  constructor() {
+    super();
+    this.message =
+      'Este agendamento não consta como Pendente, logo não é possível rejeitá-lo.';
```

Comment:
Como o nome da exceção é genérico, você não pode colocar uma mensagem específica de rejeição. A mensagem também tem que ser genérica, daí essa exceção pode ser reutilizada por outras rotas:
```suggestion
      'O agendamento deve estar com status Pendente para realizar esta ação.';
```